predictsignauxfaibles/make_list.py

```diff
@@ -0,0 +1,141 @@
+# pylint: disable=all
```

Comment:
I don't think disabling pylint for the whole module if a good idea 😕

Reply:
Also I think these functions could go in `utils.py` but I'm not against having them in a separate module :)

Reply:
See comment above:

> It looks like pylint fails when inspecting function merge_models due to some RecursionError that I have yet to investigate in detail:
> RecursionError: maximum recursion depth exceeded while calling a Python object
> Before we investigate that in-depth, disabling pylint checks for function merge_models
> 
> The following issue seems to point to operations on pandas DataFrame as a source of recursion error from pylint, but no solution was offered yet:
> volkamerlab/opencadd#63
> Unfortunately, pylint has removed in the v2.0.0 an option --optimize-astthat provided a clean workaround. The only remaining workaround seems to be to downgrade package astroid (see PyCQA/pylint#3982 (comment)). If we have time, maybe we couls comment on this last issue...
> 
> In the meantime, we are stuck with having function merge_models uncovered by linting.
> Because # pylint: disable=all disables pylint for the entire file rather than for a single function, let's leave merge_models as the only function in a module predictsignauxfaibles.merge_models.py, with pylint disabled.

See commit `66518bb2038d28960c47eecf98c809418bc01808`